tamagui/tamagui
# update expo to 49

- https://blog.expo.dev/expo-sdk-49-c6d398cdf740 went through this
- ran `npx expo-doctor@latest`
- remove eas extra, simulator does run w/o project id.
- change project place holder, no longer valid
- fixed all issues in mono repo deps
- tested on ios and web
